sktime/classification/interval_based/_tsf.py

```diff
@@ -13,7 +13,7 @@
 from joblib import delayed
 from sklearn.base import clone
 from sklearn.ensemble._forest import ForestClassifier
```

Comment:
seems like this commit should be removed as this change is reverted in the next commit... unless im missing something..